cypress/plugins/index.js

```diff
@@ -17,6 +17,6 @@
  */
 // eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
```

Comment:
It's a redundant file for cypress v10. Please see here: https://docs.cypress.io/guides/references/migration-guide#Plugins-File-Removed